src/github/deployment.ts

```diff
@@ -11,6 +11,7 @@ import { LoggerWithTarget } from "probot/lib/wrap-logger";
 export default async (context: CustomContext, jiraClient, _util, githubInstallationId: number): Promise<void> => {
 
 	if (await booleanFlag(BooleanFlags.USE_SQS_FOR_DEPLOYMENT, false, jiraClient.baseURL)) {
+		console.log("SCHEDULING MESSAGE");
```

Comment:
Console.log 😮

Reply:
at least that proves that I've run the code  :)